omnibus/resources/chef-workstation/msi/source.wxs.erb

```diff
@@ -51,144 +51,204 @@
+							<!-- Note that this refers to a Source relative to the package build directory,
+										 and not in the zip file which contains all of the chef-workstation build -->
+								<File Id="StartChefWSScript" Source="Resources\assets\start-chefws.ps1" KeyPath="yes" />
+							</Component>
+						</Directory>
+						<Directory Id="PSMODULES" Name="modules" >
+							<Component Id="ChefPSModulePath" Guid="{FA4B6948-087F-4460-A741-47E5A5E3B235}" >
+								<Environment Id="ChefPSModulePathEnvironment"
+									Name="PSModulePath" Action="set" Part="last" System="yes" Value="[PSMODULES]" />
+							</Component>
+							<Component Id="ChefPowerShellRegistryEntries" Guid="{CA712E00-B031-424D-A439-838454AF94C5}">
+								<RegistryKey Root="HKCR" Key="Chef.PowerShell">
+									<RegistryValue Type="string" Value="Chef.PowerShell" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="Chef.PowerShell\CLSID">
+									<RegistryValue Type="string" Value="{9008CA83-83E4-41FF-9C07-696E2CC47B52}" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}">
+									<RegistryValue Type="string" Value="Chef.PowerShell" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\InprocServer32">
+									<RegistryValue Type="string" Value="[System64Folder]mscoree.dll" />
+									<RegistryValue Type="string" Name="ThreadingModel" Value="Both" />
+									<RegistryValue Type="string" Name="Class" Value="Chef.PowerShell" />
+									<RegistryValue Type="string" Name="Assembly" Value="Chef.PowerShell, Version=1.0.14.0, Culture=neutral, PublicKeyToken=7def9f799d039a95" />
+									<RegistryValue Type="string" Name="RuntimeVersion" Value="v4.0.30319" />
+									<RegistryValue Type="string" Name="CodeBase" Value="[PSMODULES]chef\Chef.PowerShell.dll" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\InprocServer32\1.0.0.0">
+									<RegistryValue Type="string" Name="Class" Value="Chef.PowerShell" />
+									<RegistryValue Type="string" Name="Assembly" Value="Chef.PowerShell, Version=1.0.14.0, Culture=neutral, PublicKeyToken=7def9f799d039a95" />
+									<RegistryValue Type="string" Name="RuntimeVersion" Value="v4.0.30319" />
+									<RegistryValue Type="string" Name="CodeBase" Value="[PSMODULES]chef\Chef.PowerShell.dll" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\ProgId">
+									<RegistryValue Type="string" Value="Chef.PowerShell" />
+								</RegistryKey>
+								<RegistryKey Root="HKCR" Key="CLSID\{9008CA83-83E4-41FF-9C07-696E2CC47B52}\Implemented Categories\{62C8FE65-4EBB-45E7-B440-6E39B2CDBF29}" />
+							</Component>
+						</Directory>
+						<Directory Id="COMPONENTS" Name="components">
+							<Directory Id="WSAPPDIR" Name="chef-workstation-app">
```

Comment:
What is this `Directory` tag doing? Is it creating `components/chef-workstation-app` as part of the install? Or just listing it so that Wix knows about it and could (EG) delete it on uninstall?